mlflow/utils/autologging_utils/__init__.py

```diff
@@ -521,12 +521,24 @@ def autologging_is_disabled(integration_name):
     return False
 
 
+def is_autolog_supported(integration_name):
+    """
+    Whether the specified autologging integration is supported by the current environment.
+
+    Args:
+        integration_name: An autologging integration flavor name.
+    """
+    # NB: We don't check for the presence of autolog() function as it requires importing
+    #   the flavor module, which may cause import error or overhead.
+    return "autologging" in _ML_PACKAGE_VERSIONS.get(integration_name, {})
+
+
 def get_autolog_function(integration_name: str) -> Optional[Callable[..., Any]]:
     """
     Get the autolog() function for the specified integration.
     Returns None if the flavor does not have an autolog() function.
     """
-    flavor_module = getattr(mlflow, integration_name, None)
+    flavor_module = importlib.import_module(f"mlflow.{integration_name}")
```

Comment:
note: `getattr(mlflow, integration_name)` doesn't handle the case where flavor name contains a comma e.g., `pyspark.ml`